### ISSUE TITLE
NoClip

### ISSUE THREAD

**Issue by SquirrelJr** [SIsilicon/WorldEdit-BE#233]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem

Having no collisions while in creative mode would make editing easier “noclip”

### Proposed Solution

Java worldedit had/has this feature. Is it possible to add to the settings block in the kit? Or add as a “ ;noclip “ ?

### Alternatives

Enable cheats

**Comment by raphifou9992** [comment]
Just go into spectator mode and creative mode :/  but ye it would be usefull

**Comment by LilyTheCuteCatgirl** [comment]
its currently not posible inminecraft bedrock but ill still keep this opened if it at some point changes and becomes posible

**Comment by SIsilicon** [comment]
Yes as of writing, you can only do this with spectator mode, editor mode (Windows Only) or mods
